### PR TITLE
an attempt to fix the radio issue

### LINKED ISSUE #7546: bug with the locking of the Headset uplink after the emerging of PR#7363

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
i completely forgot about this huh

so uhh yea im making it literally not unlock ever i do not know tgui at all

## Why It's Good For The Game
closes #7546 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
tweak: radio uplink now autolocks
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
This has no testing evidence, and leaves commented code in.

Try locking it on `ui_close`

File: code/datums/components/uplink.dm [hunk 1]
```diff
@@ -326,7 +326,7 @@
 		if(failsafe_code && findtext(lowertext(message), lowertext(failsafe_code)))
 			failsafe()
 		return
-	locked = FALSE
+	// locked = FALSE
```
Comment (Archanial) [inline code/datums/components/uplink.dm]: Why are you leaving this commented?

**Comment by itsmeow** [comment]
I'm not sure the original issue exists, I can't even reproduce it. Using the headset just opens the uplink, as expected. Not the normal menu.

**Comment by etherware-novice** [comment]
FINE! iTS FINE the ill just cjklrose

**Pull request closed without merging**